resources/views/checkout/partials/sections/success/products.blade.php

```diff
@@ -1,3 +1,51 @@
 <x-rapidez-ct::card.inactive>
```

Comment:
I dont think the ordered products need a card wrapped around it, since that isn't like the design